### ISSUE TITLE
Feature request: --plopen

### ISSUE THREAD

**Issue by jwiegley** [ledger/ledger#1876]
When the `--gains` option is used, it calculates gains for a commodity across the entire history. However, it does not answer the question "Has my current position made a profit?"

This is a relatively simple question to answer:

For any commodity "10 FOO {$200.00}", simply subtract it from the current market price for FOO. If that were $250, then this commodity would be replaced by ($50 * 10).

I imagine this should be done as a reduction function in balance.cc. That is, given a balance and a set of commodity prices, reduce the balance as much as possible based on available pricing information.

**Comment by tbm** [comment]
I agree this is need, although I'm not sure I like the name `--plopen`.

**Comment by jwiegley** [comment]
"P/L Open" is what my own broker uses for this figure, meaning "Profit/Loss from position opening" as opposed to "P/L Day" or "P/L Year".
